src/kohlrahbi/unfoldedahb/unfoldedahbtable.py

```diff
@@ -212,10 +212,17 @@ def from_ahb_table(cls, ahb_table: AhbTable, pruefi: str):
                     )
                 )
 
+        metadata = ["" or None, "" or None]
```

Comment:
Denke es wäre schöner wenn wir hier ein Dictionary einführen um nicht mit Indizes arbeiten zu müssen.
```suggestion
        metadata = {"name": "" or None, "communication_direction": "" or None}
```

und es entsprechend unten anpassen.